docs/faq.rst

```diff
@@ -814,6 +814,34 @@ computation at runtime. For example:
 For more information on runtime callbacks and examples of their use,
 see `External callbacks in JAX`_.
 
+Why do some CUDA libraries fail to load/initialize?
+---------------------------------------------------
+
+When resolving dynamic libraries, :code:`ld.so` prefers in order:
+:code:`RPATH, LD_LIBRARY_PATH, RUNPATH`. JAX sets :code:`RPATH` to point to the
+JAX-relative location of the pip-installed NVIDIA CUDA packages, preferring them
+if installed. Otherwise JAX searches :code:`/lib, /usr/lib`, library paths included
```

Comment:
It might be clearer to just link to the `ld.so` man page here:
https://man7.org/linux/man-pages/man8/ld.so.8.html

(Not sure whether that's a good place to link to it.)

It is more accurate to say that "The dynamic loader `ld.so` searches ...". JAX doesn't do anything in particular.

Reply:
Alright I've trimmed it a bit, deferring most of the explanation to the man page, focusing on JAX's setting of `RPATH` and the user's role in manually setting `LD_LIBRARY_PATH`